python/lsst/ctrl/stats/report.py

```diff
@@ -0,0 +1,350 @@
+#
```

Comment:
The newly added python/lsst/ctrl/stats/report.py appears to be worthy of some summary description at the top -- basically expand upon "Report various statistics" -- to provide more context/information about the reporting.